multiapps-controller-process/src/test/java/org/cloudfoundry/multiapps/controller/process/steps/CreateOrUpdateStepWithExistingAppTest.java

```diff
@@ -202,8 +201,11 @@ private void prepareClient(CloudApplication application, Set<CloudRoute> routes,
         when(client.getApplicationProcess(application.getGuid())).thenReturn(ImmutableCloudProcess.builder()
                                                                                                   .command(command)
                                                                                                   .diskInMb(disk == null ? 1024 : disk)
-                                                                                                  .memoryInMb(memory == null ? 1024 : memory)
-                                                                                                  .healthCheckType(hcType == null ? HealthCheckType.PORT : HealthCheckType.valueOf(hcType.toUpperCase()))
+                                                                                                  .memoryInMb(memory == null ? 1024
+                                                                                                      : memory)
+                                                                                                  .healthCheckType(hcType == null
+                                                                                                      ? HealthCheckType.PORT
+                                                                                                      : HealthCheckType.valueOf(hcType.toUpperCase()))
```

Comment:
Is this a formatter change?

Reply:
Yes, the lines were automatically changed by the formatter.